app/models/saved_claim_group.rb

```diff
@@ -21,6 +21,13 @@ class SavedClaimGroup < ApplicationRecord
 
   scope :child_claims_for, ->(parent_id) { where(parent_claim_id: parent_id).where.not(saved_claim_id: parent_id) }
 
+  # Claim submission statuses
+  PENDING = 'pending' # default - waiting to be processed
+  ACCEPTED = 'accepted' # submitted to Sidekiq jobs and/or services
+  FAILURE = 'failure' # submission failed
+  PROCESSING = 'processing' # reached service, waiting for decision
+  SUCCESS = 'success' # submission succeeded
```

Comment:
What was the thought behind not having these as a lookup table rather than defined in code?